package.json

```diff
@@ -23,6 +23,7 @@
     "hexo-s3-deploy": "meteor/hexo-s3-deploy#e36f6d046660040878bb5bedfc8422f526a78be6"
   },
   "scripts": {
+    "test":"hexo clean; hexo generate",
```

Comment:
Use `&&` rather than `;`?

Reply:
In retrospect, yes.  When I first started this adventure of updating all the repos, this command had been a lot more complicated, and it wasn't clear to me whether or not `clean` returned an error code for any meaningful, expected reason.  I don't think it does, and I think any failure at this point would probably be notable enough to switch this to `&&`.

I will follow-up with a ninja commit which makes that change across all the repos.